packages/lms/src/lib/components/navigation/Entity.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import { drawerStore } from '@skeletonlabs/skeleton';
+	import type { EntityMeta } from '@mollify/types';
+	export let entity: EntityMeta;
+	let open = false;
+    let parent = entity.children.length ? true : false;
+
+    function drawerClose(): void {
+        drawerStore.close();
+    }
+	function toggle() {
+		open = !open;
+	}
+</script>
+
+<nav class="entity">
+	<div class="entity-inner">
+		<div class="entity-header hover:bg-primary-hover-token rounded-container-token p-2">
+			{#if parent}
+				<i class="icon-f">insert_drive_file</i>
```

Comment:
i wonder if a folder icon makes more sense for a parent?

Reply:
Perhaps some sort of building or academic cap icon could be used for the top level icon if its an institute, then use a book/folder for content below. I also wouldn't mind seeing some icons to differentiate lessons and assignments maybe a page icon.

Reply:
![Screenshot 2023-09-11 193634](https://github.com/Fermain/-mollify/assets/91462616/fc5c5bdd-717c-4bce-8e1a-e52a0e0ecc2b)

Reply:
Yes, i so agree! Added icons to each different type located in the index.d.ts under types.